src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    private static String command;
+    private static String description;
+    private static Task task;
+    private static boolean isTask;
+    private static boolean isCommand;
+    private static final TaskList list = new TaskList();
+
+
     public static void main(String[] args) {
+
+        welcomeMessage();
+
+        while (true) {
+            String userInput = askForInput();
+            checkTypeOfInstruction(userInput);
+            splitInput(userInput);
+
+            if (isCommand) {
+                executeCommand();
+            } else if (isTask) {
+                createTask();
+                list.addToList(task);
+            }
+
+            if (command.equals("list") || command.equals("done")) {
+                continue;
+            }
+            printSingleTask(list.getTaskCount()-1);
+            printNoOfTasks();
+        }
+    }
+
+    public static void welcomeMessage() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println(logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    private static String askForInput() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Please enter a task:");
+
+        Scanner input = new Scanner(System.in);
+        String userInput = input.nextLine();
+        return userInput;
+    }
+
+    public static void checkTypeOfInstruction(String input) {
+        isCommand = false;
+        isTask = false;
+        String[] slicedInput = input.split(" ", 2);
+        switch(slicedInput[0]) {
+            case "list":
+            case "bye":
+            case "done":
+                isCommand = true;
+                break;
+            default:
+                isTask = true;
+                break;
```

Comment:
I think you should define constants like "list" or "done" or "bye" in an enumeration